third_party/libpg_query/src_backend_parser_scan.cpp

```diff
@@ -487,7 +486,7 @@ struct yy_buffer_state
 	/* Number of characters read into yy_ch_buf, not including EOB
 	 * characters.
 	 */
-	yy_size_t yy_n_chars;
+	int yy_n_chars;
```

Comment:
I wonder where this and other changes came from? What made this change to an int?

Reply:
I believe that this happened when re-generating the parser code after changes to `select.y` and `create.y`. It normally looks much worse, like in my PR for the list lambdas where I did bigger changes to the parser (https://github.com/duckdb/duckdb/pull/3694/files).

Reply:
I am aware this is generated, maybe this is down to generator versions

Reply:
I did generate the files myself now and the `int` is indeed caused by an older(?) bison version.
```
tania@laptop duckdb % /usr/bin/bison --version
bison (GNU Bison) 2.3
```
@lokax I will open a PR to your branch with the newly generated files.

Reply:
> I did generate the files myself now and the `int` is indeed caused by an older(?) bison version.
> 
> ```
> tania@laptop duckdb % /usr/bin/bison --version
> bison (GNU Bison) 2.3
> ```
> 
> @lokax I will open a PR to your branch with the newly generated files.

Thanks, but my version seems to be newer?.
```text
bison (GNU Bison) 3.5.1
```

Reply:
Yes, I was suspecting that. It is weird that a newer version uses `int` though...